internal/host/installcmd.go

```diff
@@ -136,6 +143,23 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 	return []*models.Step{step}, nil
 }
 
+func getDiskFormatCmds(log logrus.FieldLogger, host models.Host) (string, error) {
+	var inventory models.Inventory
+	if err := json.Unmarshal([]byte(host.Inventory), &inventory); err != nil {
+		log.Errorf("Failed to get inventory from host with id %s", host.ID)
+		return "", err
+	}
+	formatCmds := ""
+	for _, disk := range inventory.Disks {
+		if disk.Bootable {
+			dev := GetDeviceFullName(disk.Name)
+			formatCmds = formatCmds + fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=33 ;", dev)
+			formatCmds = formatCmds + fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=33 seek=$((`blockdev --getsz %s` - 33)); ", dev, dev)
```

Comment:
What about wipefs and 'sgdisk -Z'?  Do we have those on RHCOS?

Reply:
Yes, there are available in RHCOS.
Do you think we need them too?

Reply:
It's in the ironic code, I think it's worth adding

Reply:
Sure